pkg/common/cns-lib/volume/manager.go

```diff
@@ -469,10 +483,39 @@ func (m *defaultManager) createVolumeWithImprovedIdempotency(ctx context.Context
 		// Local instance of CreateVolume details that needs to
 		// be persisted.
 		volumeOperationDetails *cnsvolumeoperationrequest.VolumeOperationRequestDetails
-		// Information related to volume created by CNS
-		resp *CnsVolumeInfo
+		// quotaInfo consists of values required to populate QuotaDetails in CnsVolumeOperationRequest CR.
+		quotaInfo     *cnsvolumeoperationrequest.QuotaDetails
+		clusterFlavor cnstypes.CnsClusterFlavor
 	)
 
+	if extraParams != nil {
```

Comment:
Need to add podVMOnStretchSupervisor FSS check ?

Reply:
Added the FSS check below this code.